nca/NetworkConfig/NetworkConfigQuery.py

```diff
@@ -861,6 +881,20 @@ def split_to_tcp_and_non_tcp_conns(conns):
 
         return tcp_conns, non_tcp_conns
 
+    @staticmethod
+    def _exclude_dns_entries_from_peer_list(peers_list):
+        """
+        Return a peer list with no DNSEntry peers
+        :param list peers_list: list of peers pairs
+        :rtype: list
+        """
+        peers_list_wo_dns = []
+        for peers_pair in peers_list:
+            # a DNSEnty peer may only be in the dest (peers_pair[1])
+            if not isinstance(peers_pair[1], DNSEntry):
+                peers_list_wo_dns.append(peers_pair)
+        return peers_list_wo_dns
+
```

Comment:
consider instead return from `allowed_conns` only TCP for a connection to `serviceEntry`

Reply:
done